boot/bootutil/src/bootutil_priv.h

```diff
@@ -403,7 +403,26 @@ boot_img_sector_off(const struct boot_loader_state *state, size_t slot,
 #endif  /* !defined(MCUBOOT_USE_FLASH_AREA_GET_SECTORS) */
 
 #ifdef MCUBOOT_RAM_LOAD
-#define IMAGE_RAM_BASE ((uintptr_t)3)
+#   ifdef __BOOTSIM__
```

Comment:
extra spaces, also in line 423-425

Reply:
The spaces are intentional? I'm not sure why people insist on not indenting preprocessor directives.

Reply:
ACK.